### PR TITLE
fix: a grammer error

### PR DESCRIPTION
**Pull request by apsinghdev**
## Description

In this PR, I've fixed a small grammar error.

## Related Issue

Fixes: #1315 

## Changes Made

It replaces `"wont"` with `"won't"` and `"Only your signing experience is"` with `"Only your signing experience will be"`

## Checklist

- [x] I have tested these changes locally and they work as expected.
- [x] I have added/updated tests that prove the effectiveness of these changes.
- [x] I have updated the documentation to reflect these changes, if applicable.
- [x] I have followed the project's coding style guidelines.
- [x] I have addressed the code review feedback from the previous submission, if applicable.

## Summary by CodeRabbit

- **Bug Fixes**
	- Updated text in the Document Share Button for enhanced clarity and grammatical accuracy.

### CODE REVIEW

File: packages/ui/components/document/document-share-button.tsx [hunk 1]
```diff
@@ -142,8 +142,8 @@ export const DocumentShareButton = ({
           <DialogTitle>Share your signing experience!</DialogTitle>
 
           <DialogDescription className="mt-4">
-            Don't worry, the document you signed or sent wont be shared; only your signing
-            experience is. Share your signing card and showcase your signature!
+            Don't worry, the document you signed or sent won't be shared; only your signing
```
Comment (catalinpit) [inline packages/ui/components/document/document-share-button.tsx:145]: I'd change it altogether to something like this:

```
Rest assured, your document is strictly confidential and will never be shared. Only your signing experience will be highlighted. Share your personalized signing card to showcase your signature!
```

Comment (apsinghdev) [reply packages/ui/components/document/document-share-button.tsx]: Sounds great! Updated.

**Comment by apsinghdev** [comment]
Here is how it looks now.
grammer-er

**Review by catalinpit — APPROVED** [review]
Thank you!

**Pull request merged**